### ISSUE TITLE
feat: page for calculation of PoB items price to set by base owner.

### ISSUE THREAD

**Issue by dd84ai** [darklab8/fl-darkrelay#1]
Karst — Today at 3:08 PM
I've made a powerful spreadsheet which contains all the commodities needed to build any POB equipment:
https://discoverygc.com/forums/showthread.php?tid=205587
Players can copy the sheet and substitute their own values, so they can manage their equipment prices.
Fab — Today at 4:03 PM
if @Darkwind The Dark Duck got freetime he can make something similar but automated with the ever changing econ prices and recipes
Darkwind The Dark Duck — Today at 4:05 PM
Same thoughts visited me when watching it. Same thoughts. 😋 
Fab — Today at 4:05 PM
jammi has an econ update lined up which will change prices and sell points I believe
Karst — Today at 4:06 PM
Potentially yes but this would depend on the prices a POB owner wishes to set being entirely automated in some way.
I don't think that is realistic, as I have a number of factors that led to my pricing beyond bare flcomp econ, such as connectivity, ease of ID access, difficulty/distance and such.
You could generate automated prices but I'd probably never use them myself, I'd still want to manually adjust them as I see fit.
Darkwind The Dark Duck — Today at 4:07 PM
mm yeah.. it is bothersome to make tool fitting Pob owner all usage cases here
like ideally table should be accounting for time to deliver goods to specific location of a base owner
as generic we can just count only item counts requirement to build smth (and potentially add sum calculator what needs to be in total) 
Karst — Today at 4:08 PM
And that's not considering commodities that quite simply cannot have a generated price, like Earhart mats or Prototype components.
Fab — Today at 4:08 PM
use base value for that
or perhaps a manual override
Karst — Today at 4:09 PM
Well that would just return values that are useless. Those commodities make up the bulk of the price on a large portion of these items.
Darkwind The Dark Duck — Today at 4:09 PM
at the end of a day this table is probably good enough for what it does... its only downfall being not very automated in updates
automating the same is bothersome anyway
Karst — Today at 4:10 PM
I would very much hope that there are in fact update notes whenever recipe requirements change.
Darkwind The Dark Duck — Today at 4:10 PM
Although we could write smth like... autogenerated table for export to excel => insert prices on your own as desired 😄
Or even no need for insert.. we can calculate prices on our own
Simple solution

TLDR: literally coding exactly this table just not in excel 
Karst — Today at 4:11 PM
Mm
A lot of these are "safe", I guess
Like all the common code ingredients, I doubt those are ever going to change, or not unless the entire commodity list is culled again
Darkwind The Dark Duck — Today at 4:12 PM
Very niche tool at the end of a day... it doesn't look as good to automate it i think perhaps

People would have benefited from easy findable just recipes/items requirement and where to buy them navigation

Price calculation itself for pob owners is too much niche tool 
Karst — Today at 4:14 PM
Of course, I thought about how this could be automated. I just don't know if you'd want to.
Like, in Ionic's case, it's a plat exporter base.
If there's a really good Plat route to one of the things it buys, I'd want to pay less for that. Because that just makes sense, the route is already good.
Whereas for example, I pay a ludicrous overprice for Superconductors because the route is kinda but not really short, and has absolutely no first leg trade good.
It's simply easier to do the prices manually for POBs, imo.
Automating connectivity is uh
Yeah, probably not happening.
Darkwind The Dark Duck — Today at 4:16 PM
uh. automating such stuff just requires server side to implement it. (different user input and based on that making decisions) 
there is gigabytes amount of data to connect with each other (if accounting for trade routes connectivity) 
it is impossible to implement in any neat way at client side through static site rendered site
because it would just take too much data to load to user browser to do so at client
and i would not make going from Static Site Generator concept i have currently
because current deisgn is very comfy to keep up to date and host for free 😋
therefore... doing otherwise is making sense only as separate tool, which is entirely different effort 
Karst — Today at 4:18 PM
I suppose in some technical way, you could input the exact location of the POB, have the script gather all information from any base around in addition to what the base makes itself which in turn is dependent on the location, and somehow create a profit formula for a two-way trip for every commodity, which would still be affected by hard to quantify factors?
Eeeeh
Honestly, some things are just better done manually.
Darkwind The Dark Duck — Today at 4:19 PM
Sure... it is easy to make technically, just going to be server side requiring application if doing within known to me limitation
Bothersome 😄
Darkwind The Dark Duck — Today at 4:32 PM
Even if we would be desiring just rendering this table automatically
it woul dnot be having this neat ordering and grouping as u have 
but that's probably not a big issue
Darkwind The Dark Duck — Today at 4:58 PM
Even if we implement this table antoupdatable in pure client side
i am bothered by the problem of having too many buttons 😄
The top menu with buttons is having too many of them as it is :/ somehow more buttons to fit will be needed then 
Darkwind The Dark Duck — Today at 7:13 PM
i think i know an elegant solution to my buttons problem
....well.. having button to show or hide extra buttons 😄 
and i don't need to change any look from the current one, it can be the same flexbox train
i will just move some non essential buttons into extra category that will be hidden by default 
On another hand if i will not be inventing more tabs, then many buttons problem is not a problem hmm 
Urgh. You can solve with buttons any problem except problem of having too many buttons
Ash — Today at 7:31 PM
maybe instead of trying to automate everything, you could export instead only the closes buy point from a solar location (input by user)
then user can use formulas to import from your excell tab and modify values by percentage or whatver, as they see fit
so base name, buy price,sell price time to travel there (in seconds?)
Darkwind The Dark Duck — Today at 7:35 PM
maybe instead of trying to automate everything, you could export instead only the closes buy point from a solar location (input by user)
That's the problem. Input by user.
therefore we need dynamic recalculation of trade routes 
therefore we need server side
therefore we build separate app
Ash — Today at 7:36 PM
silly question, you already do that for trade routes?
Darkwind The Dark Duck — Today at 7:37 PM
nope. Still static site generator 😄
i just pregenerate 2 gigabytes of micro pages, that user clicks through 
Ash — Today at 7:37 PM
ahh
now i get it 😄
Darkwind The Dark Duck — Today at 7:39 PM
Tools desiring server side get to be accumulated, may be to record as idea for future to do actually
